yt_dlp/extractor/tiktok.py

```diff
@@ -112,7 +111,8 @@ def _call_api(self, ep, query, video_id, fatal=True,
                 self._WORKING_APP_VERSION = (app_version, manifest_app_version)
                 self.write_debug('Imported app version combo from extractor arguments')
             elif app_version or manifest_app_version:
-                self.report_warning('Only one of the two required version params are passed as extractor arguments', only_once=True)
+                self.report_warning('Only one of the two required version params are passed as extractor arguments',
+                                    only_once=True)
```

Comment:
revert unnecessary/unrelated formatting change
```suggestion
                self.report_warning('Only one of the two required version params are passed as extractor arguments', only_once=True)
```